tests/phpunit/AbstractBaseTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeTransformation\Tests;
+
+use Keboola\SnowflakeDbAdapter\Connection;
+use Keboola\SnowflakeTransformation\Exception\ApplicationException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Serializer\Encoder\JsonEncode;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+
+abstract class AbstractBaseTest extends TestCase
+{
+    protected string $dataDir = '/data';
+
+    protected Connection $connection;
+
+    public function setUp(): void
+    {
+        $databaseConfig = $this->getDatabaseConfig();
+        $this->connection = $this->getConnection($databaseConfig['workspace']);
+    }
+
+    protected function getDatabaseConfig(): array
+    {
+        return [
+            'workspace' => [
+                'host' => $this->getEnv('SNOWFLAKE_HOST'),
+                'port' => $this->getEnv('SNOWFLAKE_PORT'),
+                'warehouse' => $this->getEnv('SNOWFLAKE_WAREHOUSE'),
+                'database' => $this->getEnv('SNOWFLAKE_DATABASE'),
+                'schema' => $this->getEnv('SNOWFLAKE_SCHEMA'),
+                'user' => $this->getEnv('SNOWFLAKE_USER'),
+                'password' => $this->getEnv('SNOWFLAKE_PASSWORD'),
+            ],
+        ];
+    }
+
+    protected function putConfig(array $config): void
+    {
+        $jsonEncode = new JsonEncode();
+        $json = $jsonEncode->encode($config, JsonEncoder::FORMAT);
+        file_put_contents($this->dataDir . '/config.json', $json);
```

Comment:
toto zavani datadir testama, ten dir by mel byt urcite v tempu, protoze takhle je persistentni a to je nahovno (kdyz ti v testu vypadne putConfig, tak test bude vesele fungovat, pokud tam ten config zustane z predchoziho testu), 
takze bud temp, anebo rovnou datadir testy (idelane)

Reply:
tak jsem to předělal na datadir testy a s tím jsem musel downgradovat phpunit na v 7.x protože to datadir testy vyžadují a poslal jsem PR na update balíčků https://github.com/keboola/datadir-tests/pull/23

Reply:
zupdatoval jsem datadir testy a php-component tak už jsem tu mohl použít latests balíčky